src/WebJobs.Extensions.DurableTask/ContextImplementations/DurableClient.cs

```diff
@@ -563,6 +597,19 @@ async Task<PurgeHistoryResult> IDurableOrchestrationClient.PurgeInstanceHistoryA
             return await this.DurabilityProvider.PurgeInstanceHistoryByInstanceId(instanceId);
         }
 
+        /// <inheritdoc />
+        async Task<IList<PurgeHistoryResult>> IDurableOrchestrationClient.PurgeInstanceHistoryAsync(IEnumerable<string> instanceId)
+        {
+            IList<PurgeHistoryResult> purgehistoryresultList = new List<PurgeHistoryResult>();
+            foreach (string id in instanceId)
+            {
+                PurgeHistoryResult result = await this.DurabilityProvider.PurgeInstanceHistoryByInstanceId(id);
```

Comment:
Should we do these purge operations in parallel instead of one-at-a-time? Otherwise, I worry that we're not adding much value with this new API.

Reply:
Hi, Chris. Thanks for your review. I have updated already and had a small question about the output. 

The Current output for this function is a list of PurgeHistoryResult. This class only have one attribute, InctanceDeleted, meaning the number of deleted instances. So, the return for this overloaded method looks like this: ["1", "1,","1,"........]. In this way, the advantage is that customer will directly know which instance is deleted and which one is not. 

Another option for the output is just return an instance of PurgeHistoryResult. This will look more concise. The number reflects the number of deleted instances, resulting to the unknown of which one not deleted. But when reading the code, I think as long as the instance ID is valid, it will be deleted successfully. Considering this, I am unsure if the "Purge History unsuccessfully" should be concerned. 

I am unsure which one is better. I would appreciate it if you could give me some advice. :)